### DISCUSSION TITLE
How to insert Images, where is the Image toolbar button?

### DISCUSSION THREAD

**Discussion by kal60** [gsantner/markor#1750] [Q&A] [answered]
Hi!
I'm using Markor 2.9 on a Samsung Galaxy Tab S7+ with Android 12. When I create a Markup-File I can't find the Toolicon to add Images (if I add an image as markup-text it's displayed).
![AnnotiumPhotoAnnotation_1657139237](https://user-images.githubusercontent.com/8725410/177638293-1b3cf64c-c31e-4151-9719-76d28505daa1.jpg)

This is how it's looking on my tablet:

![Screenshot_20220706-212830_Markor](https://user-images.githubusercontent.com/8725410/177638687-b99b67a2-d86d-4dca-a48d-d2dd0d4dd8c9.jpg)

Regards
Knut

**Comment by gsantner** [comment, marked as answer]
Attachment (clippy) icon

**Reply by kal60** [reply]
Oh, great! I did not used that before. Thanks for your very quick help :-)

**Reply by gsantner** [reply]
no problem
